perl/bin/bystro-server.pl

```diff
@@ -71,7 +71,13 @@
 
 say "Running Annotation queue server";
 
-my $BEANSTALKD_TIMEOUT = 60;
+my $BEANSTALKD_RESERVE_TIMEOUT = 10;
+# Needs to be larger than RESERVE_TIMEOUT
```

Comment:
How large is RESERVE_TIMEOUT? Is that client side?

Reply:
The reserve timeout is client side yeah. Basically what will happen is that we try to put the job, and if we don't get back a response in 10 seconds, we stop trying.

For context, a typically put request takes 1-2 milliseconds, so most cases that takes > 10 seconds probably mean the connection died or the beanstalkd server is dead

Reply:
Before we attempt to reserve however, we attempt to connect to the server. If the connection fails, the put command will also fail, with a "timeout" error.

If connection succeeds, but the server dies during reservation, I think we can still have an issue. That chance is small though, since it would have to happen in the millisecond before the reserve command was received. It may also be possible for the issue to happen during reservation, but 10 seconds is short enough that that is unlikely (and AWS load balancer TTL default is 60 seconds)

Reply:
I could however wrap the reserve command in a way that makes it time out client side to prevent stalling